qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -11,6 +11,10 @@
     filesize = 10
     host_script = serial_host_send_receive.py
     guest_script = VirtIoChannel_guest_send_receive.py
+    ppc64, ppc64le:
+        virtio_dev_iommu_platform = on
+        virtio_dev_disable_legacy = on
+        virtio_dev_disable_modern = off
```

Comment:
Serial can be launched without IOMMU configuration, so this would be a new case/variant.

Reply:
do you mean we will still keep both original one and new one at the same time ?